### PR TITLE
DEEPVARIANT: add a gzi input field to avoid the problem for compressed fasta file

### PR DESCRIPTION
**Pull request by gq1**
…sed fasta.gz file

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/nf-core/deepvariant/test.yml [hunk 1]
```diff
@@ -19,3 +19,12 @@
...
     - path: output/deepvariant/versions.yml
+
+- name: deepvariant test_deepvariant_no_fasta_gzi
+  command: nextflow run ./tests/modules/nf-core/deepvariant -entry test_deepvariant_no_fasta_gzi -c ./tests/config/nextflow.config
+  tags:
+    - deepvariant
+  exit_code: 1
+  stdout:
+    contains:
+      - "Failed to open FASTA index genome.fasta.gz.fai: No such file or directory"
```
Comment (FriederikeHanssen) [inline tests/modules/nf-core/deepvariant/test.yml:30]: can you also add a test with the gzi?

Comment (muffato) [reply tests/modules/nf-core/deepvariant/test.yml:30]: It looks like there is already one here: https://github.com/nf-core/test-datasets/tree/modules/data/pangenomics/homo_sapiens

Comment (gq1) [reply tests/modules/nf-core/deepvariant/test.yml:30]: In this directory, no any bam or cram files based on this reference.

Comment (FriederikeHanssen) [reply tests/modules/nf-core/deepvariant/test.yml:30]: Has the test been added? I don't see it 🙈

Comment (gq1) [reply tests/modules/nf-core/deepvariant/test.yml]: @FriederikeHanssen 
Thanks for merging the test data, and the new tests added here now.
Could you please review it?

Comment (gq1) [reply tests/modules/nf-core/deepvariant/test.yml]: @FriederikeHanssen Do you have time to look at this extra tests?
Thanks.

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍 All tests passed.

**Pull request merged**